src/lib/dongle/message.rs

```diff
@@ -250,7 +251,7 @@ impl Response for WalletPublicKey {
         // parse as a secp pubkey, and rust-bitcoin will do the right thing
         // (since it understands the bip32 spec as just using "keys" and
         // always encoding them compressedly).
-        let pk = bitcoin::secp256k1::PublicKey::from_slice(&data[1..1 + pk_len])?;
+        let pk = bitcoin::secp256k1::PublicKey::from_slice(&data[1..=1 + pk_len])?;
```

Comment:
Are you aware that you are changing the behaviour here to enlarge the amount of data being sliced?

Reply:
Derp, no, this is not intentional! Thanks.

Reply:
This was not even supposed to be committed because it is clearly wrong.